### DISCUSSION TITLE
How to update Flowise version deployed on AWS?

### DISCUSSION THREAD

**Discussion by blissiraDC** [FlowiseAI/Flowise#1109] [Q&A]
Hi, as per the title, I have flowise 1.3.6 deployed on AWS, what is the procedure to update to 1.3.8? 

Any help is greatly appreciated.

**Comment by blissiraDC** [comment]
Thanks for your help....

**Comment by Jaredude** [comment]
I know this is an old post, but figured I'd put this here in case anyone is searching for the answer

ssh into your EC2 instance and then do the following:

cd Flowise/docker
docker-compose stop
docker pull flowiseai/flowise
docker-compose up -d

**Reply by zbalsara21** [reply]
Thank you so much. I was searching 10 different posts before got this. Truly appreciate.
